website/versioned_docs/version-0.5/navigation.md

```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
 
 ## React Navigation
 
-The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code.
+The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code. Assuming you are on the latest react-native version.
```

Comment:
Can you rephrase it? React Navigation works with some older versions. What's the point of this sentence? It sounds a bit strange.

Reply:
sorry i missed this one to remove from my commit 🙏,  it just a local change test because it's my first time trying to contribute here 🙏 

8a25db7d4468a4a82a24348d15d20ff2eeb3c1b2